pyo3_special_method_derive_macro/src/lib.rs

```diff
@@ -576,7 +581,7 @@ pub fn auto_debug(input_stream: TokenStream) -> TokenStream {
 ///     pub phone_number: String,
 /// }
 /// ```
-#[proc_macro_derive(Getattr, attributes(pyo3_smd))]
+#[proc_macro_derive(Getattr, attributes(skip))]
```

Comment:
IMO this ones needs the no_skip as well! More often than not private attributes get public getters!

Reply:
Yeah, makes sense.